### PR TITLE
Add 'CleanUpBuild' tag to kill process build steps

### PR DESCRIPTION
**Pull request by blindpirate**
So that we can filter them out easily.

### CODE REVIEW

File: .teamcity/src/main/kotlin/common/extensions.kt [hunk 1]
```diff
@@ -212,7 +212,7 @@ fun BuildType.killProcessStep(stepName: String, daemon: Boolean) {
             executionMode = BuildStep.ExecutionMode.ALWAYS
             tasks = "killExistingProcessesStartedByGradle"
             gradleParams =
-                buildToolGradleParameters(daemon).joinToString(separator = " ")
+                (buildToolGradleParameters(daemon) + "-Dscan.tag.CleanUpBuild").joinToString(separator = " ")
```
Comment (wolfs) [inline .teamcity/src/main/kotlin/common/extensions.kt:215]: We have a `buildScanTag()` method for doing this.

**Comment by big-guy** [comment]
This is a great idea @blindpirate

File: .teamcity/src/main/kotlin/common/extensions.kt (hunk 1 above)
Comment (blindpirate) [reply .teamcity/src/main/kotlin/common/extensions.kt]: Thanks! Good to know!

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Review by wolfs — COMMENTED** [review]
LGTM!

**Pull request merged**